### ISSUE TITLE
Example in README to install hemibrainr doesn't work

### ISSUE THREAD

**Issue by mmc46** [natverse/natmanager#14]
https://github.com/natverse/natmanager/blob/master/README.Rmd

``` r
natmanager::install('flyconnectome/hemibrainr')
#> Error in match.arg(collection): 'arg' should be one of "core", "natverse"
```

**Comment by jefferis** [comment]
@SridharJagannathan Marta is right, this need fixing. Would you mind picking this up quickly?

**Comment by SridharJagannathan** [comment]
Closed via 85deb52d165f38296328dafa8566611b8457ff7d
